src/main/java/com/codeborne/selenide/drivercommands/CreateDriverCommand.java

```diff
@@ -7,14 +7,14 @@
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.support.events.EventFiringWebDriver;
 import org.openqa.selenium.support.events.WebDriverEventListener;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 import java.util.List;
-import java.util.logging.Logger;
```

Comment:
Looks like we need to extend checkstyle `IllegalImport` rule with jul logging to avoid it's using